packages/apps/tools/src/components/Global/PredKeysSelect/index.tsx

```diff
@@ -1,20 +1,19 @@
 import type { ISelectProps } from '@kadena/react-ui';
 import { FormFieldWrapper, Select } from '@kadena/react-ui';
+import type { TPredicate } from '@kadena/types';
```

Comment:
I don't think we have the convention of prefixing types with `T`. 
So why not just use `Predicate`?

Reply:
I prefer without the prefix